sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/implementation/NettyUtility.java

```diff
@@ -98,18 +101,20 @@ static void validateNettyVersions(Consumer<String> logger) {
         String nettyTcnativeVersion = pomVersions.get(NETTY_TCNATIVE_VERSION_PROPERTY);
 
         Map<String, Version> nettyVersions = Version.identify();
-        List<String> versionMismatches = new ArrayList<>(11); // There are 11 Netty dependencies in the pom.xml file.
+        List<String> versionInformation = new ArrayList<>(11); // There are 11 Netty dependencies in the pom.xml file.
 
+        Set<String> nonNativeNettyVersions = new HashSet<>();
         for (String artifact : REQUIRED_NETTY_VERSION_ARTIFACTS) {
             Version version = nettyVersions.get(artifact);
 
             // Version shouldn't be null as azure-core-http-netty has it as a dependency, but it could have been
             // excluded. Include it as a warning.
             if (version == null) {
-                versionMismatches.add("'io.netty:" + artifact + "' version not found (expected: " + nettyVersion + ")");
-            } else if (!Objects.equals(version.artifactVersion(), nettyVersion)) {
-                versionMismatches.add("'io.netty:" + artifact + "' version: " + version.artifactVersion()
-                    + " (expected: " + nettyVersion + ")");
+                versionInformation.add("'io.netty:" + artifact + "' (not found and is required)");
```

Comment:
Before doing all this computation, should we first check if logging at `info` level is enabled?